bezpecnostni-doporuceni-open-source.md

```diff
@@ -109,6 +109,15 @@ je repozitář se zdrojovým kódem označen jako neudržovaný (např. funkcí
 
 Pokud se jedná o knihovnu zveřejněnou ve správci balíčků, je knihovna takto označena i v tomto správci.
 
+### O.12 Součástí repozitáře je soubor CONTRIBUTING
+Aby byla kooperace s přispívajícími vývojáři co nejjednodušší, měl by repozitář obsahovat soubor `CONTRIBUTING.md`. Jeho obsah by měl čtenáři objasnit především:
+
+- Nastavení a zprovoznění prostředí pro běh kódu.
```

Comment:
Jiný typ odrážek než ve zbytku dokumentu – tzn. použít raději `*`

K diskusi je asi gramatika – použití vět (velké písmeno, tečka) namísto výčtu (první velké písmeno, pak malé, čárka a nakonec tečka). Osobně preferuji použití vět (jako je to tady).